mlrun/datastore/google_cloud_storage.py

```diff
@@ -12,44 +12,82 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import json
+import os
 from pathlib import Path
 
 from fsspec.registry import get_filesystem_class
+from google.auth.credentials import Credentials
+from google.cloud.storage import Client, transfer_manager
+from google.oauth2 import service_account
 
 import mlrun.errors
 from mlrun.utils import logger
 
-from .base import DataStore, FileStats, makeDatastoreSchemaSanitizer
+from .base import DataStore, FileStats, make_datastore_schema_sanitizer
 
 # Google storage objects will be represented with the following URL: gcs://<bucket name>/<path> or gs://...
 
 
 class GoogleCloudStorageStore(DataStore):
     using_bucket = True
+    workers = 8
+    chunk_size = 32 * 1024 * 1024
 
     def __init__(self, parent, schema, name, endpoint="", secrets: dict = None):
+        self._storage_client = None
+        self._storage_options = None
         super().__init__(parent, name, schema, endpoint, secrets=secrets)
```

Comment:
```suggestion
        super().__init__(parent, name, schema, endpoint, secrets=secrets)
        self._storage_client = None
        self._storage_options = None
```
convention is to call `super().__init__` first, unless there's a reason to do otherwise.